### PR TITLE
Add configurationGet and registerMultiChannelReportListener methods

### PR DESCRIPTION
**Pull request by RobinBol**
Add configurationGet method, add registerMultiChannelReportListener method, add customSaveMessage string.

### CODE REVIEW

File: lib/zwave/ZwaveDevice.js [hunk 1]
```diff
@@ -302,6 +299,34 @@ class ZwaveDevice extends MeshDevice {
...
+		// Check if device has command class
+		const commandClassConfiguration = this.getCommandClass('CONFIGURATION');
+		if (commandClassConfiguration instanceof Error ||
+			typeof commandClassConfiguration.CONFIGURATION_GET !== 'function') {
+			this.error('Missing COMMAND_CLASS_CONFIGURATION');
+			return Promise.reject(new Error('missing_command_class_configuration'));
+		}
+
+		return new Promise((resolve, reject) => {
+			commandClassConfiguration.CONFIGURATION_GET({
```
Comment (WeeJeWel) [inline lib/zwave/ZwaveDevice.js]: Returns a promise already

File: lib/zwave/ZwaveDevice.js [hunk 2]
```diff
@@ -302,6 +299,34 @@ class ZwaveDevice extends MeshDevice {
 		});
 	}
 
+	/**
+	 * Method that retrieves the value of a configuration parameter from the node.
+	 * @param {Object} options
+	 * @param {number} options.index - Parameter index
+	 * @returns {*}
+	 */
+	configurationGet(options = {}) {
```
Comment (WeeJeWel) [inline lib/zwave/ZwaveDevice.js]: add `async`

File: lib/zwave/ZwaveDevice.js [hunk 3]
```diff
@@ -318,7 +343,8 @@ class ZwaveDevice extends MeshDevice {
 	 * @private
 	 */
 	_parseSetting(settingObj = {}, value) {
-		let parser, customParser
+		let parser,
+			customParser;
```
Comment (WeeJeWel) [inline lib/zwave/ZwaveDevice.js]: Where is `customParser` used?

File: lib/zwave/ZwaveDevice.js [hunk 4]
```diff
@@ -753,26 +810,40 @@ class ZwaveDevice extends MeshDevice {
...
- 						case 2: payload.Properties1['Key Attributes'] = 'Key Held Down'; break;
- 						case 3: payload.Properties1['Key Attributes'] = 'Key Pressed 2 times'; break;
- 						case 4: payload.Properties1['Key Attributes'] = 'Key Pressed 3 times'; break;
- 						case 5: payload.Properties1['Key Attributes'] = 'Key Pressed 4 times'; break;
- 						case 6: payload.Properties1['Key Attributes'] = 'Key Pressed 5 times'; break;
- 					}
- 				}
- 			}
+			// Catch central scene echos and (sometimes) failing parser
+			if (command.name === 'CENTRAL_SCENE_NOTIFICATION') {
```
Comment (WeeJeWel) [inline lib/zwave/ZwaveDevice.js]: Why is this re-added?

File: lib/zwave/ZwaveDevice.js (hunk 1 above)
Comment (RobinBol) [reply lib/zwave/ZwaveDevice.js]: Fixed

File: lib/zwave/ZwaveDevice.js (hunk 2 above)
Comment (RobinBol) [reply lib/zwave/ZwaveDevice.js]: Fixed

File: lib/zwave/ZwaveDevice.js (hunk 3 above)
Comment (RobinBol) [reply lib/zwave/ZwaveDevice.js]: A couple of lines down, in the same `_parseSetting` method.

File: lib/zwave/ZwaveDevice.js (hunk 4 above)
Comment (RobinBol) [reply lib/zwave/ZwaveDevice.js]: It is not re-added (only code style changed), it was never gone, see my latest comment here: https://github.com/athombv/node-homey-meshdriver/pull/66

**Pull request merged**